src/_events/2020-pete-gallagher-talk/summary.md

```diff
@@ -0,0 +1,2 @@
+In this talk, learn the basics of the Dot Net Framework, and using the raspberry pi.
```

Comment:
```suggestion
In this talk, learn the basics of the Dot NET Framework, and using it with the raspberry pi.
```

Reply:
Should it be .NET rather than Dot Net? .NET is what Microsoft has on their website?